pkg/common/k8s/tracing_test.go

```diff
@@ -0,0 +1,55 @@
+package k8s
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+	"knative.dev/eventing-kafka/pkg/common/kafka/constants"
+	injectionclient "knative.dev/pkg/client/injection/kube/client"
+	logtesting "knative.dev/pkg/logging/testing"
+	"knative.dev/pkg/system"
+	tracingconfig "knative.dev/pkg/tracing/config"
+	"os"
+	"testing"
+)
+
+// Test The LoggingContext() Functionality
+func TestInitializeTracing(t *testing.T) {
+
```

Comment:
copy paster comments - need to be updated ; ). this is not a logging test

Reply:
oops, fixed